### PR TITLE
Issue 361

### PR DESCRIPTION
**Pull request by Elina297**
I put the github versions of packages mia, miaViz and bluster. I also removed ggord as Giulio said that it is no more needed

### CODE REVIEW

**Comment by antagomir** [comment]
This somehow overlaps with #371  - let us handle that first and then sync with this before this can be merged.

**Comment by Elina297** [comment]
Yes strange. I can also see my commits from the other branch for issue 365, I think that shouldn't be the case. Do you know what might be the cause of this problems ?

**Comment by antagomir** [comment]
You have opened two different PRs. 
#371 and #372 

Can you either remove one, combine them, or confirm if both are essential?

**Comment by Elina297** [comment]
One is for the issue 361, to install github versions of packages instead of bioc versions. The other one is for issue 365 concerning the references to data sets in the exercises. So I would say that both are "essentials". But I don't know how to combine them

**Comment by antagomir** [comment]
no need to combine but let us complete one of them first..

File: 98_exercises.Rmd [hunk 1]
```diff
@@ -266,8 +266,8 @@ row trees, respectively.
 Try to extract some of the other TreeSE elements listed in chapter \@ref(containers).
```
Comment (antagomir) [inline 98_exercises.Rmd:266]: Remove "Try to"

File: 98_exercises.Rmd [hunk 2]
```diff
@@ -245,8 +245,8 @@ got stuck, you can refer to chapter \@ref(assay-slot) of this book.
 
 ### Feature information
 
-1. Import the mia package, load peerj13075 with `data` and store it into a
-   variable named `tse`.
+1. Import the mia package, load any of the example data sets mentioned in Chapter 3.3
+   with `data` and store it into a variable named `tse`.
```
Comment (antagomir) [inline 98_exercises.Rmd:249]: remove "with data" here and other similar places

File: 98_exercises.Rmd [hunk 3]
```diff
@@ -354,8 +355,9 @@ Useful functions: nrow, ncol, dim, summary, table, quantile, unique, transformAs
 
 ### Alternative experiments
 
-1. Import the mia package, load GlobalPatterns with `data` and store it into a
-   variable named `tse`.
+1. Import the mia package, load one of the example data sets mentioned in Chapter 3.3
+   with `data` (you need one with taxonomic information) and store it into 
+   a variable named `tse`.
 2. Check the taxonomic ranks of the features with `taxonomyRanks`. What is the
```
Comment (antagomir) [inline 98_exercises.Rmd:361]: Add here question: "Can you find the same taxonomyRanks from `rowData`?"

**Review by antagomir — APPROVED** [review]
Ok with the requested minor changes.

**Comment by antagomir** [comment]
Can you resolve the conflicts, then update the PR?

**Comment by antagomir** [comment]
Could you resolve the open discussion points and update the PR? We could then merge.

**Comment by antagomir** [comment]
Can you make a fresh clone, add your changes and open a new PR? Might be easier.

**Comment by antagomir** [comment]
@himmil  could you transfer this into a new PR and merge the relevant parts, taking into account the feedback on this PR?

**Comment by antagomir** [comment]
@himmil can you assign this to Noah? I do not know their github username - just to open a new PR and migrate the relevant parts from here, with the suggested changes. This should be manually but should be relatively straightfwd. This PR should be closed then, and referred to in the new PR.

**Comment by Insaynoah** [comment]
I created a new PR with the adjustments.

**Comment by antagomir** [comment]
Can you @ake123 close this one if it can be closed (if nothing to do with this any more). 

Just make sure to also take into account the feedback that was given in the comments.

Please close issues when done, and if not sure, then follow-up and check if it can be closed.

**Pull request closed without merging**